### PR TITLE
Makes voice of god a even bigger force to be reckoned with

### PR DESCRIPTION
**Pull request by WarRage333**
## Changelog
:cl:

balance: Voice of god is now way stronger

/:cl:

## Why It's Good For The Game

adds farting and screaming to voice of god

### CODE REVIEW

**Review by yoyobatty — CHANGES_REQUESTED** [review]
Modularise this or place a header and trailer comment indicating it's a hippie edit.

**Comment by WarRage333** [comment]
done didly done

**Comment by Manly-Dwarf** [comment]
Did you fix Voice of God spam automuting people?

**Comment by yoyobatty** [comment]
That's probably better to put it another PR, maybe even upstream.

**Comment by Carbonhell** [comment]
> Did you fix Voice of God spam automuting people?

/atom/movable/proc/say(message, bubble_type, var/list/spans = list(), sanitize = TRUE, datum/language/language = null, ignore_spam = FALSE, forced = null)

to fix, just add ignore_spam = TRUE to any say() calls in the code forced by some object (example: voice of god)

**Comment by mikolaj007** [comment]
+1 sure why not

**Comment by AnthonyJNgo** [comment]
Honestly, voice of god shouldn't be immune to spam, you shouldn't be saying SCREAM 20 times in a row anyways. +1.

**Review by yoyobatty — APPROVED** [review]

File: code/modules/surgery/organs/vocal_cords.dm [hunk 1]
```diff
@@ -251,6 +251,8 @@
 	var/static/regex/clap_words = regex("clap|applaud")
 	var/static/regex/honk_words = regex("ho+nk") //hooooooonk
 	var/static/regex/multispin_words = regex("like a record baby|right round")
+	var/static/regex/fart_words = regex("fart|poo|poop|defecate") //hippie edit
```
Comment (ghost) [inline code/modules/surgery/organs/vocal_cords.dm:254]: can be modularized by adding them under `hippiestation/code/modules/surgery/organs/vocal_cords.dm`

File: code/modules/surgery/organs/vocal_cords.dm [hunk 2]
```diff
@@ -562,6 +564,24 @@
 			var/mob/living/L = V
 			L.SpinAnimation(speed = 10, loops = 5)
 
+	// hippie start -- FART
```
Comment (ghost) [inline code/modules/surgery/organs/vocal_cords.dm:567]: can just make a seperate proc that does like
```
else
 hippie_voice_of_god(message)
```

**Review by ghost — CHANGES_REQUESTED** [review]

File: code/modules/surgery/organs/vocal_cords.dm (hunk 1 above)
Comment (WarRage333) [reply code/modules/surgery/organs/vocal_cords.dm:254]: Tried to but didn't work on my server when I tested it,  prolly me just being bad tho

**Comment by MaterialisticThings** [comment]
+1 only if we can fix it from muting you

**Comment by lunaman22** [comment]
+1 AAAAAAAAAAAAAAAAA

**Comment by WarRage333** [comment]
I really don't have the know-how how to fix these properly if anyone wants to take over this feel free

**Pull request closed without merging**